recipe/meta.yaml

```diff
@@ -47,11 +47,16 @@ requirements:
     - python-dateutil
     - pyyaml
     - rasterio >=1.3.2
-    - sqlalchemy
+    - singledispatch
+    - sqlalchemy <2.0
     - geoalchemy2
     - xarray >=0.9,!=2022.6.0
+    - celery >=4
+    - paramiko
+    - redis-py
+    - sshtunnel
```

Comment:
@SpacemanPaul these are optional and I would leave them out they will blow up minimal install quite a bit and aren't really used by most.

Reply:
I don't think I added them - I think they were added by conda-forge-bot (whose PR I cloned) - I only updated the SQLAlchemy pin.